### PR TITLE
DTFS2-3483—fe size page

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: gef-ui/templates/partials/about-exporter.njk [hunk 1]
```diff
@@ -32,11 +37,156 @@
...
+              text: "No",
+              checked: isFinanceIncreasing === 'false',
+              attributes: {
+                'data-cy': 'is-financing-increasing-radio-no'
+              }
+            }
+          ]
+        }) }}
+
+        poo {{ isFinanceIncreasing }}
```
Comment (ttbarnes) [inline gef-ui/templates/partials/about-exporter.njk:169]: ?

Comment (monkemedia) [reply gef-ui/templates/partials/about-exporter.njk]: Hee hee - removed :)

File: gef-ui/server/controllers/about-exporter/index.js [hunk 2]
```diff
@@ -1,18 +1,86 @@
...
+      aboutExporterErrors.push({
+        errRef: 'probabilityOfDefault',
+        errMsg: 'Enter the probability of default',
+      });
+    }
+
+    if (!body.isFinanceIncreasing) {
+      aboutExporterErrors.push({
+        errRef: 'isFinanceIncreasing',
+        errMsg: 'Enter theSelect whether your financing to the exporter is increasing as a result of this new GEF facility ',
```
Comment (ttbarnes) [inline gef-ui/server/controllers/about-exporter/index.js:51]: typo

File: gef-ui/server/controllers/about-exporter/index.js [hunk 3]
```diff
@@ -1,18 +1,86 @@
...
+  try {
+    const { exporterId } = await api.getApplication(applicationId);
+    const { details } = await api.getExporter(exporterId);
+
+    if (aboutExporterErrors.length > 0) {
+      return res.render('partials/about-exporter.njk', {
+        errors: validationErrorHandler(aboutExporterErrors),
+        industries: details.industries || null,
+        smeType: body.smeType || null,
+        probabilityOfDefault: body.probabilityOfDefault || null,
```
Comment (ttbarnes) [inline gef-ui/server/controllers/about-exporter/index.js:65]: probably don't need to do the `|| null` part ?

**Review by ttbarnes — APPROVED** [review]
Minor comment, otherwise LGTM

**Comment by monkemedia** [comment]
@ttbarnes thanks tony. Changes have been made

**Pull request merged**